res/PyTorchExamples/examples/interpolate/__init__.py

```diff
@@ -0,0 +1,26 @@
+import torch
+import torch.nn as nn
+
+
+# model
+class net_interpolate(nn.Module):
+    def __init__(self, scale_factor):
+        super().__init__()
+        self.scale_factor = scale_factor
+
+    def forward(self, input):
+        return torch.nn.functional.interpolate(
+            input,
+            scale_factor=self.scale_factor,
+            mode='bilinear',
+            align_corners=True,
+            recompute_scale_factor=True)
+
+    def onnx_opset_version(self):
```

Comment:
This operation requires minimum 11 onnx opset version, but fails on version 13+ during onnx-to-tf conversion (because some operation is not supported yet in tf_onnx 1.9)

Reply:
maybe add the condition as a comment?

Reply:
Added above